sw/host/opentitantool/src/command/version.rs

```diff
@@ -0,0 +1,33 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::{Result};
+use erased_serde::Serialize;
+use std::any::Any;
+use structopt::StructOpt;
+
+use opentitanlib::app::command::CommandDispatch;
+use opentitanlib::app::TransportWrapper;
+use opentitanlib::version::BUILD_GIT_VERSION;
+
+#[derive(Debug, StructOpt)]
+pub struct Version {
+}
+#[derive(Debug, serde::Serialize)]
+pub struct VersionResponse {
+    version: String,
+}
+
+
+impl CommandDispatch for Version {
+    fn run(
+        &self,
+        _context: &dyn Any,
+        _transport: &TransportWrapper,
+    ) -> Result<Option<Box<dyn Serialize>>> {
+        Ok(Some(Box::new(VersionResponse {
+            version: BUILD_GIT_VERSION.to_string(),
```

Comment:
Up to you, but I also like to include the BUILD_SCM_STATUS flag in the version output.  If the status is `clean`, the binary should be reproducible.  If the status is `modified`, you know the build came from a WIP or other "dirty" sandbox.  

Since the output is structured json, I'd probably just add a bool to the version response struct.

Reply:
Sure, why not.

Reply:
Sure, why not.